core/server/services/themes/middleware.js

```diff
@@ -38,31 +37,18 @@ themeMiddleware.ensureActiveTheme = function ensureActiveTheme(req, res, next) {
     }
 
     next();
-};
+}
 
-// ### Update Template Data
-// Updates handlebars with the contextual data for the current request
-themeMiddleware.updateTemplateData = function updateTemplateData(req, res, next) {
+function updateGlobalTemplateOptions(req, res, next) {
```

Comment:
Specific fn name, hopefully removes need for comments

Reply:
Really in favor of such changes :smile: We have some more places where the comment is rather unnecessary